### ISSUE TITLE
ForegroundScale is being parsed with the culture of the machine.

### ISSUE THREAD

**Issue by ADD-David-Antolin** [unoplatform/uno.resizetizer#96]
## Current behavior

The value is parsed with the culture of the regional configuration of the machine. 

## Expected behavior

The value is parsed with invariant culture. So "0.65" is 65%.

## How to reproduce it (as minimally and precisely as possible)

Assuming your regional configuration is `English - United States` (or another in which the symbol of the decimal separator is the dot).

1. Create a new project with the Uno Visual Studio Extension.

2. In `base.props` check `UnoIcon` item has `ForegroundScale="0.65"`.

3. Launch Windows head, the icon should be seen properly in the taskbar.

4, Change machine regional configuration to `Spanish - Spain`.

5. Sign out & sign back in for changes to apply machine wide.

6. Open the solution and rebuild the windows head.

7. Launch Windows head, the icon in the taskbar is a blank background.

8. In `base.props` change `ForegroundScale` value to `0,65` (with a comma).

9. Rebuild the windows head.

10. Launch Windows head, the icon should be seen properly in the taskbar.

## Environment

Nuget Package:

Package Version(s): 1.0.2

Affected platform(s):

- [X] iOS
- [X] Android
- [X] WebAssembly
- [X] WebAssembly renders for Xamarin.Forms
- [X] Windows
- [X] Build tasks

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [X] 2022 (version: 17.5.2)
- [ ] for Mac (version: )

## Anything else we need to know?

I think the problem is located [here](https://github.com/unoplatform/uno.resizetizer/blob/e23dfa53da77e48510173e99a29c260b08b826f5/src/Resizetizer/src/ResizeImageInfo.cs#L118). It should use the overload with `IFormatProvider `and pass `CultureInfo.InvariantCulture`.

**Comment by pictos** [comment]
Not sure if it's a bug... Like on C# even if your language is other than English you can't specify `0.65` as `0,65`, right? That should propagate to MSBuild tasks as well.

@nickrandolph  or @jeromelaban thoughts?

**Comment by ADD-David-Antolin** [comment]
> Not sure if it's a bug... Like on C# even if your language is other than English you can't specify `double x = 0.65` as `double x = 0,65`, right? That should propagate to MSBuild tasks as well.
> 
> @nickrandolph or @jeromelaban thoughts?

It does not make any sense that two programmers, one in the States and one in Spain have to be continually modifying the value just because the have different regional formats in the machine. It is bad for maintainability.

I know <UnoIcon> is based in <MauiIcon>. They have done the same fix I propose in: https://github.com/dotnet/maui/pull/11965

**Comment by ADD-David-Antolin** [comment]
> @ADD-David-Antolin thanks for the report. Could you [provide a binlog](https://platform.uno/docs/articles/uno-builds-troubleshooting.html) of your build?

@Suriman will upload it.

**Comment by jeromelaban** [comment]
@ADD-David-Antolin Indeed, if the issue is fixed in let's replicate the same fix for sure. Thanks for the lookup @ADD-David-Antolin!
